scripts/globals/abilities/pets/dream_shroud.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------
+-- Dream Shroud
+---------------------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+require("scripts/globals/utils");
+
+---------------------------------------------------
+
+function onAbilityCheck(player, target, ability)
+    return 0,0;
+end;
+
+function onPetAbility(target, pet, skill, summoner)
+    local bonusTime = utils.clamp(summoner:getSkillLevel(SKILL_SUM) - 300, 0, 200);
+    local duration = 180 + bonusTime;
+    local hour = VanadielHour();
+    local buffvalue = 0;
+	
+	if hour == 0 then
+	     buffvalue = 13;
```

Comment:
Got mixed tabs and spaces in file. ~~Pick one please.~~ DSP wants spaces. 4 spaces = 1 indent.